book/make_docs.nu

```diff
@@ -8,8 +8,10 @@ if $book_exists == false {
   exit --now
 }
 
-# Clean all commands and regenerate the docs
-do -i { rm book/commands/*.md }
+# Old commands are currently not deleted because some of them
+# are platform-specific, and a single run of this script will not regenerate
```

Comment:
Looks good to me now, just one last question: about 'some of them are platform-specific', can you list which ones？

Reply:
It came up awhile ago, but `registry query` is Windows-only, and a few others (can't remember which) are POSIX-only.

Reply:
I will register all the necessary plugins and run `source book/make_docs.nu` to generate the docs, that won't miss the docs of plugins